src/kiwoong/ManagementSystemImpl.java

```diff
@@ -0,0 +1,92 @@
+package kiwoong;
+
+import kiwoong.ManagementSystem;
+import kiwoong.Member;
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class ManagementSystemImpl implements ManagementSystem {
+
+    private Map<Integer, Member> memberlist = new HashMap<>();
+
+    @Override
+    public boolean addMember(String name, int age, int id) {
+
+        if(memberlist.containsKey(id)) {return false;}
+
+        Member newMember = new Member(name, age, id);
+        memberlist.put(id, newMember);
+        return true;
+    }
+
+    @Override
+    public boolean removeMember(int id) {
+
+        if (memberlist.containsKey(id)){
+            memberlist.remove(id);
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean updateMember(int id, String name) {
+
+        if (memberlist.containsKey(id)){
+            Member members = memberlist.get(id);
+            members.setName(name);
+        }
+        return false;
+    }
+
+    @Override
+    public boolean updateMember(int id, int age) {
+        if (memberlist.containsKey(id)){
+            Member members = memberlist.get(id);
+            members.setAge(age);
+        }
+        return false;
+    }
+
+    @Override
+    public boolean updateMember(int id, String name, int age) {
+        if (memberlist.containsKey(id)){
+            Member members = memberlist.get(id);
+            members.setName(name);
+            members.setAge(age);
+        }
+        return false;
+    }
+
+    @Override
+    public Member findById(int id) {
+        if (memberlist.containsKey(id)){
+            return memberlist.get(id);
+        }
+        return null;
+    }
```

Comment:
```suggestion
    @Override
    public Member findById(int id) {
        return memberlist.get(id)
    }
```

`Map.get()` 메서드는 Map 내부에 해당 Key를 지닌 요소가 없다면 `null`을 리턴합니다.
따라서 코드를 위와 같이 수정할 수 있습니다!

Reply:
감사합니다!!!